service/history/workflow/mutable_state_impl.go

```diff
@@ -4566,7 +4543,7 @@ func (e *MutableStateImpl) createCallerError(
 	return serviceerror.NewInvalidArgument(msg)
 }
 
-func (_ *MutableStateImpl) unixNanoToTime(
+func (*MutableStateImpl) unixNanoToTime(
```

Comment:
move this into `common/primitives/timestamp` or `common/util`?

Reply:
3 usages of this function are:
```golang
e.unixNanoToTime(now.UnixNano())
```
:facepalm: